Portal/UIKit/PortalUIApplication.swift

```diff
@@ -17,6 +18,7 @@ public enum UIApplicationMessage {
     case didBecomeActive(application: UIApplication)
     case willTerminate(application: UIApplication)
     case remoteNotification(RemoteNotification, application: UIApplication)
+    case handle(transaction: SKPaymentTransaction, queue: SKPaymentQueue)
```

Comment:
I would rename this to `handlePaymentTransaction(SKPaymentTransaction, queue: SKPaymentQueue)` to be consistent with the rest of the enum's cases.